src/mumble/GlobalShortcut_unix.cpp

```diff
@@ -4,6 +4,7 @@
 // Mumble source tree or at <https://www.mumble.info/LICENSE>.
 
 #include "GlobalShortcut_unix.h"
+#include "GlobalShortcut_xdp.h"
```

Comment:
We only include this, if DBus support is built into Mumble. Therefore, this include (and the functionality below) should probably be guarded
```suggestion
#ifdef USE_DBUS
#  include "GlobalShortcut_xdp.h"
#endif
```